### PR TITLE
User sees the newest posts at the top of the feed

### PR DESCRIPTION
**Pull request by salindholm**
[User sees the newest posts at the top of the feed](https://www.pivotaltracker.com/story/show/176613673)

## User Story
```
As a an API,            
In order to provide the client with the newest posts,         
I want to be able to sort by created date.
```

## Changes proposed in this pull request:
* Adding sort by newest first in index action
* Removing validations for image and preview
* Creating seeds for user and adding user_id to each post

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -4,9 +4,14 @@
...
-Post.create(track: "Cookie Chips", artists: 'Rejjie Snow', image: 'https://i.scdn.co/image/ab67616d0000b273af865889727087bf5652aae3', preview: 'https://p.scdn.co/mp3-preview/244c7855b611950bd8843e61fd5ef5f41bd919f6?cid=9165f2ed52ac4632b2c23038c2fbe1d9', description: 'I listen to this while Im eating Cookie and Chips')
-Post.create(track: "CELLZ", artists: 'MF DOOM', image: 'https://i.scdn.co/image/ab67616d0000b2731c74e2608e3caae82a1680bd', preview: 'https://p.scdn.co/mp3-preview/3a26b78e35731d64d068ed11d1b456cf47fafe21?cid=9165f2ed52ac4632b2c23038c2fbe1d9', description: 'This song has a one minute and 45 seconds intro!')
+User.create(email: 'user1@mail.com', password: 'password', password_confirmation: 'password' )
+User.create(email: 'user2@mail.com', password: 'password', password_confirmation: 'password' )
+User.create(email: 'user3@mail.com', password: 'password', password_confirmation: 'password' )
+User.create(email: 'user4@mail.com', password: 'password', password_confirmation: 'password' )
```
Comment (ViktorHek) [inline db/seeds.rb]: we don't have a signup function so `password_confirmation` isn't necessary

File: db/seeds.rb [hunk 2]
```diff
@@ -4,9 +4,14 @@
...
+
+Post.create(user_id: 1, track: 'All I Want for Christmas Is You', artists: 'Mariah Carey', image: 'https://i.scdn.co/image/ab67616d0000b2734246e3158421f5abb75abc4f', preview: 'https://p.scdn.co/mp3-preview/bbafd15ff484394a0ca106d5fef0a81eeea4ef5b?cid=9165f2ed52ac4632b2c23038c2fbe1d9', description: 'This is the best christmas song ever! All time favorite!')
```
Comment (ViktorHek) [inline db/seeds.rb]: please remove this song. I have listened to it too many times

File: db/seeds.rb (hunk 1 above)
Comment (salindholm) [reply db/seeds.rb]: Noted, its removed now!

File: db/seeds.rb (hunk 2 above)
Comment (salindholm) [reply db/seeds.rb]: Okey :)

**Pull request merged**